### PR TITLE
use trusty only for hhvm

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

**Comment by dbu** [comment]
is this a regression with symfony 3.3? see also https://github.com/symfony-cmf/seo-bundle/pull/330 which seems to have random errors with sf 3.3 too

**Comment by ElectricMaxxx** [comment]
Seems so, but shall we support 3.3 as an unstable release at all?

**Comment by ElectricMaxxx** [comment]
not that random, i think. Did a rebuild and the error occurs again. Do we know something about the error handling and exception controller on symfony 3.3?

**Comment by ElectricMaxxx** [comment]
there have been several [changes](https://github.com/symfony/twig-bundle/commits/3.3/Resources/views/Exception/exception.html.twig) I think we have to require 3.3 lokaly and make the test less sensitive.

File: .travis.yml [hunk 1]
```diff
@@ -12,17 +10,18 @@ cache:
...
 matrix:
   include:
     - php: 7.1
-      env: DEPS=dev SYMFONY_VERSION=3.3.*
+    - php: 7.0
+      env: SYMFONY_VERSION=3.2.*
     - php: 5.6
       env: SYMFONY_VERSION=2.8.* COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 7.1
-      env: SYMFONY_VERSION=3.1.*
```
Comment (dbu) [inline .travis.yml]: 3.1 is EOL. i suggest we test 2.8, 3.2 and 3.3

**Pull request merged**